pythonforandroid/recipes/feedparser/__init__.py

```diff
@@ -4,7 +4,7 @@
 class FeedparserPyRecipe(PythonRecipe):
     version = '5.2.1'
     url = 'https://github.com/kurtmckee/feedparser/archive/{version}.tar.gz'
-    depends = [('hostpython2', 'python3crystax'), 'setuptools']
+    depends = [('hostpython2', 'hostpython3crystax', 'hostpython3'), 'setuptools']
```

Comment:
Same here, you can remove everything but setuptools.

Reply:
addressed